### DISCUSSION TITLE
建议流水线列表页右边的过滤功能的弹窗中能支持回车键提交表单

### DISCUSSION THREAD

**Discussion by r0oot** [Tencent/bk-ci#4637] [General]
<img width="359" alt="QQ20210708-142735@2x" src="https://user-images.githubusercontent.com/1669272/124873180-b8307900-dff8-11eb-9128-2e1a3e168e70.png">
这里每天要用很多次，每次都要在这里浪费几秒钟时间，虽然不多，但是就不是很流畅，希望能支持回车键提交。

**Comment by iccengan** [comment]
#2407 已记录issue
